app/controllers/collections_controller.rb

```diff
@@ -70,17 +70,27 @@ def show
 
     if @collection.collection_preference.show_random? || params[:show_random]
       # show a random selection of works/bookmarks
-      @works = Work.in_collection(@collection).visible.random_order.limit(ArchiveConfig.NUMBER_OF_ITEMS_VISIBLE_IN_DASHBOARD).includes(:pseuds, :tags, :series, :language, :approved_collections)
```

Comment:
The old `.visible` method returns a different scope for admins (to show hidden works). With Elasticsearch we currently [never show any hidden works in any queries](https://github.com/otwcode/otwarchive/blob/e00d3b137c78306a87bc9cf63a4a47b5cca7efd1/app/models/search/work_query.rb#L120-L122). So there's a functionality change here, though I don't think it's a deal breaker.

Reply:
tbh it drives me bonkers that hidden works show in such a weird spot and nowhere else, so I consider this an improvement and not a dealbreaker.